app/src/main/java/org/kiwix/kiwixmobile/zim_manager/SimplePageChangeListener.kt

```diff
@@ -20,8 +20,10 @@ package org.kiwix.kiwixmobile.zim_manager
 
 import androidx.viewpager.widget.ViewPager.OnPageChangeListener
 
-class SimplePageChangeListener(private val onPageSelectedAction: (Int) -> Unit) :
-  OnPageChangeListener {
+class SimplePageChangeListener @JvmOverloads constructor(
```

Comment:
SimplePageChangeListener is only used from kotlin so the JvmOverloads is unecessary